projects/plugins/jetpack/extensions/plugins/post-publish-qr-post-panel/components/icons/index.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
Are there going to be more icons for this plugin? If not, maybe we can forgo the extra file and directory, and instead add the svg directly in `projects/plugins/jetpack/extensions/plugins/post-publish-qr-post-panel/index.js`?

Reply:
addressed here https://github.com/Automattic/jetpack/pull/23281/commits/06e0388bef25b7afe0df2947b4ee46ffbc918e9a